### ISSUE TITLE
Feature Request: Add option to not show Volume Flyout when the play/pause, previous or next buttons are pressed

### ISSUE THREAD

**Issue by arnavbhate** [ModernFlyouts-Community/ModernFlyouts#719]
**Is your feature request related to a problem? Please describe:**

When I am watching something, if I use the play/pause, previous or next buttons the Volume flyout pops up and blocks part of the screen, detracting from the viewing experience.

**Describe the solution you'd like:**

I would like an option to not show the Volume flyout in this case.

**Describe alternatives you've considered:**

Disabling the Volume flyout while watching something. However, it is frustrating to have to go to the settings to disable the Volume flyout (also #592).

**Additional context:**

This is not a bug as this happens with the Windows Volume flyout as well.

**Comment by Samuel12321** [comment]
So basically you want the volume flyout not to show when only the media flyout has been triggered.

**Comment by arnavbhate** [comment]
No, I am saying that no flyout should appear when the keyboard play/pause, previous or next buttons are pressed.

**Comment by Cyberdroid1** [comment]
Oh, so you want to be able to disable Windows flyouts selectively, instead of turning all the flyouts off?

**Comment by arnavbhate** [comment]
What I am meaning to say is that when I use the keyboard play/pause, previous or next buttons, a flyout appears which blocks the view of the thing I am watching. I don't want the flyout to appear when these are pressed, but I still want it to appear when I change the volume.

**Comment by eProject** [comment]
guess something like: disable on fullscreeen...

**Comment by arnavbhate** [comment]
> guess something like: disable on fullscreeen...

Ya could also be an alternative

**Comment by ShankarBUS** [comment]
Sounds good, I will look into it

**Comment by Naba8909** [comment]
Hey
